mis_builder/models/mis_report_instance.py

```diff
@@ -190,16 +203,18 @@ def _compute_dates(self):
             ("d", _("Day")),
             ("w", _("Week")),
             ("m", _("Month")),
+            ("q", _("quarterly")),
             ("y", _("Year")),
             ("date_range", _("Date Range")),
         ],
         string="Period type",
     )
-    is_ytd = fields.Boolean(
+    is_ytd = (
```

Comment:
This change is a bit weird, normally it's not necessary to put brackets around a field definition